prepare_release.py

```diff
@@ -179,7 +180,10 @@ def process_file(file_name, versions, dry_run):
 
         # Remove old file, replacing by the edited one
         os.remove(file_name)
-        os.rename(file_temp.name, file_name)
+        # os.rename() raises "[Errno 18] Invalid cross-device link" error is temp file is in a different
+        # filesystem so it is safer to copy & remove
+        shutil.copy(file_temp.name, file_name)
```

Comment:
Surprisingly, it seems there isn't any os.copy() operation, so I'm using that shutil library. Not sure if it is the best option. Suggestion welcome.